cylc/rose/stem.py

```diff
@@ -534,28 +536,28 @@ def main():
     # opts.group is stored by the --task option.
     rose_stem_options = OptionGroup(parser, 'Rose Stem Specific Options')
     rose_stem_options.add_option(
-        "--task", "--group", "-t", "-g",
+        "--task", "--group", '-t', '-g',
         help=(
-            "Specify a group name to run. Additional groups can be specified"
-            "with further `--group` arguments. The suite will then convert the"
-            "groups into a series of tasks to run."
+            "Specify a group of tasks to run. "
+            "Additional groups can be "
+            "specified with further `--group` arguments. "
+            "Rose stem adds group names to the RUN_NAMES template variable "
+            "which a workflow can use to enable groups of tasks."
         ),
         action="append",
         metavar="PATH/TO/FLOW",
         default=[],
         dest="stem_groups")
     rose_stem_options.add_option(
-        "--source", '-s',
+        "--source", "-s",
         help=(
-            "Specify a source tree to include in a rose-stem suite. The first"
-            "source tree must be a working copy as the location of the suite"
-            "and fcm-make config files are taken from it. Further source"
-            "trees can be added with additional `--source` arguments. "
-            "The project which is associated with a given source is normally "
-            "automatically determined using FCM, however the project can "
-            "be specified by putting the project name as the first part of "
-            "this argument separated by an equals sign as in the third "
-            "example above. Defaults to `.` if not specified."
+            "Specify a source tree to include in a rose-stem suite. "
+            "Defaults to \".\" if unspecified. "
+            "You can use --source more than once to "
+            "specify any number of sources. "
+            "The first --source argument must be a working "
+            "copy which should contain a workflow definition "
+            "in the rose-stem subdirectory"
```

Comment:
Minor, just keeping in style of other help items.
```suggestion
            "in the rose-stem subdirectory."
```